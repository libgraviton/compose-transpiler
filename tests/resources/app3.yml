_inheritance:
  extends: ./parentfile/subdir/app3parent.yml
  unsets:
    - "components.hans"
