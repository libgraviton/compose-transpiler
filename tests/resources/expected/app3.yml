version: 99
services:
  fred:
    image: 'fred:latest'
  memcached:
    image: memcached:latest

networks:
  internal:
    external: false
